uint32[] ids
geometry_msgs/Point[] xys
geometry_msgs/Quaternion[] oris
